src/tests/utils.cairo

```diff
@@ -1,8 +1,12 @@
+mod constants;
+
 use array::ArrayTrait;
+use array::SpanTrait;
 use core::result::ResultTrait;
 use option::OptionTrait;
-use starknet::ContractAddress;
 use starknet::class_hash::Felt252TryIntoClassHash;
+use starknet::ContractAddress;
```

Comment:
This conflicts with the import sorter. `ContractAddress` should precede `class_hash` because capital letters are smaller ASCII values ([alphabet to ASCII](https://www.easycalculation.com/unit-conversion/alphabets-ascii-code.php))

Reply:
It's probably better to discuss in import guidelines #696